monkey/infection_monkey/utils/windows/hidden_files.py

```diff
@@ -0,0 +1,59 @@
+HIDDEN_FILE = 'C:\\monkey-hidden-file'
+HIDDEN_FILE_WINAPI = 'C:\\monkey-hidden-file-winAPI'
+HIDDEN_FOLDER = 'C:\\monkey-hidden-folder'
```

Comment:
Trying to create files and folders in the `c:\\` drive is problematic for 2 reasons:
1. Surprisingly, there's no promise that there is a C drive in a Windows PC. It's completely possible to install Windows on a drive called `D:` and not have a `C:` drive at all. 
2. Writing to that specific directory requires Administrator privileges, but the technique doesn't require Admin to execute.

In summary it would be best to write to `%Temp%` or even the current directory instead of `C:\`.